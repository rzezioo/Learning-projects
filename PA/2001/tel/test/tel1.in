10 10
1 3 2 9 6 1 7 6 1 6
1 3 1 6 7 7 2 8 4 10
