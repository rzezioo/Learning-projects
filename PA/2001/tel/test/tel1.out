0000100010
0111101111
